hrms/api/__init__.py

```diff
@@ -624,3 +625,26 @@ def get_allowed_states_for_workflow(workflow: dict, user_id: str) -> list[str]:
 	return [
 		transition.state for transition in workflow.transitions if transition.allowed in user_roles
 	]
+
+# Notification
+@frappe.whitelist(methods=["GET"])
+def subscribe_push_notification(fcm_token: str):
```

Comment:
these 2 APIs could be part of FrappeNotification only. Other apps can use them to subscribe/unsubscribe

Reply:
done !